livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/beta/assistant_llm.py

```diff
@@ -81,6 +93,7 @@ def __init__(
         client: AsyncClient | None = None,
         api_key: str | None = None,
         base_url: str | None = None,
+        on_file_uploaded: OnFileUploaded | None = None,
```

Comment:
Let's follow the callback we have in VoiceAssistant, what about file_uploaded_cb ?